sources/packages/backend/libs/integrations/src/esdc-integration/fed-restriction-integration/fed-restriction.processing.service.ts

```diff
@@ -139,7 +132,7 @@ export class FedRestrictionProcessingService {
         const invalidDataMessage = restriction.getInvalidDataMessage();
         if (invalidDataMessage) {
           const errorMessage = `Found record with invalid data at line number ${restriction.lineNumber}: ${invalidDataMessage}`;
-          result.errorsSummary.push(errorMessage);
+          processSummary.error(errorMessage);
```

Comment:
This will trigger a different conversation. Yes, we can change to `warn` to avoid the retry, but it also should alert the Ministry in some way. I will change to a warn to move on and discuss further with biz.